addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1424,6 +1458,14 @@ options.registry.CarouselItem = options.Class.extend({
         this.counterEl = document.createElement('span');
         titleTextEl.appendChild(this.counterEl);
 
+        this.$el.tooltip({
+            selector: '[data-tooltip-text]',
+            placement: 'top',
+            delay: 100,
+            title: function () {
+                return this.dataset.tooltipText;
+            },
+        });
```

Comment:
I think this code should be removed, It seems it was used to show tooltips on non `".active"` sidebar tab buttons and since `data-title` was removed in this [task](https://github.com/odoo/odoo/pull/65573/commits/c71cf6c618b48da36fbc5063140703312248bc09), the code: `this.$('[data-title]').tooltip({` has no effect.

We can simply show the title for carousel options and enable/disable this using a data-tooltip attribute :

```
options.registry.CarouselItem = options.Class.extend({
...
   start: function () {
   ...
   this.$el.tooltip({
            selector: '[data-tooltip]',
            ...

```
&

`<we-button ... data-tooltip="" title="Content">`

Another solution is to add tooltips code in `_updateRightPanelContent` (in a global way) to activate tooltips after adding snippet options (`customize$Elements`) to the sidebar.

Reply:
Ok, you can make a separate commit removing the useless code and implementing a generic solution to have tooltips on elements. I don't think it should be in `_updateRightPanelContent` but simply generic on the editor panel (during its initialization, just like `[data-title]` before (I don't think the elements have to be in the DOM for this to work: you say "in this container, the elements with the selector "XXX" should have a tooltip... to test to confirm)).

For the syntax I would do it that way: simply use the `title` attribute (which is translated) and add a separate data-tooltip attribute as you suggested but which is only a boolean to indicate that the title should be shown as a BS tooltip.